internal/commands/vaultsecrets/secrets/create_test.go

```diff
@@ -45,12 +50,21 @@ func TestNewCmdCreate(t *testing.T) {
 			Name:    "Failed: No secret name arg specified",
 			Profile: testProfile,
 			Args:    []string{},
-			Error:   "ERROR: accepts 1 arg(s), received 0",
+			Error:   "ERROR: missing required flag: --data-file=DATA_FILE_PATH",
 		},
 		{
 			Name:    "Good: Secret name arg specified",
 			Profile: testProfile,
-			Args:    []string{"test"},
+			Args:    []string{"test", "--data-file=DATA_FILE_PATH"},
```

Comment:
Curious if this also supports [heredoc](https://en.wikipedia.org/wiki/Here_document) input

Reply:
I can def look at this as part of Milestone 3 (toward the project's end). That would likely be useful for many